jimp.d.ts

```diff
@@ -18,7 +41,120 @@ declare namespace Jimp {
+        chnl: number;
+    }
+
+    interface FontInfo {
+        face: string;
+        size: number;
+        bold: number;
+        italic: number;
+        charset: string;
+        unicode: number;
+        stretchH: number;
+        smooth: number;
+        aa: number;
+        padding: [number, number, number, number];
+        spacing: [number, number];
+    }
+
+    interface FontCommon {
+        lineHeight: number;
+        base: number;
+        scaleW: number;
+        scaleH: number;
+        pages: number;
+        packed: number;
+        alphaChnl: number;
+        redChnl: number;
+        greenChnl: number;
+        blueChnl: number;
+    }
+
+    interface Font {
+        chars: {
+            [char: string]: FontChar;
+        };
+        kernings: {
+            [firstString: string]: {
+                [secondString: string]: number;
+            };
+        };
+        pages: string[];
+        common: FontCommon;
+        info: FontInfo;
+    }
+
     class Jimp {
+        // Constants
+        static AUTO: -1;
+
+        // supported mime types
+        static MIME_PNG: 'image/png';
+        static MIME_JPEG: 'image/jpeg';
+        static MIME_JGD: 'image/jgd';
+        static MIME_BMP: 'image/bmp';
+        static MIME_X_MS_BMP: 'image/x-ms-bmp';
+        static MIME_GIF: 'image/gif';
+        // PNG filter types
+        static PNG_FILTER_AUTO: -1;
+        static PNG_FILTER_NONE: 0;
+        static PNG_FILTER_SUB: 1;
+        static PNG_FILTER_UP: 2;
+        static PNG_FILTER_AVERAGE: 3;
+        static PNG_FILTER_PATH: 4;
+
+        // resize methods
+        static RESIZE_NEAREST_NEIGHBOR: 'nearestNeighbor';
+        static RESIZE_BILINEAR: 'bilinearInterpolation';
+        static RESIZE_BICUBIC: 'bicubicInterpolation';
+        static RESIZE_HERMITE: 'hermiteInterpolation';
+        static RESIZE_BEZIER: 'bezierInterpolation';
+
+        // Align modes for cover, contain, bit masks
+        static HORIZONTAL_ALIGN_LEFT: 1;
+        static HORIZONTAL_ALIGN_CENTER: 2;
+        static HORIZONTAL_ALIGN_RIGHT: 4;
+
+        static VERTICAL_ALIGN_TOP: 8;
+        static VERTICAL_ALIGN_MIDDLE: 16;
+        static VERTICAL_ALIGN_BOTTOM: 32;
+
+        // Font locations
+        static FONT_SANS_8_BLACK: string;
+        static FONT_SANS_10_BLACK: string;
+        static FONT_SANS_12_BLACK: string;
+        static FONT_SANS_14_BLACK: string;
+        static FONT_SANS_16_BLACK: string;
+        static FONT_SANS_32_BLACK: string;
+        static FONT_SANS_64_BLACK: string;
+        static FONT_SANS_128_BLACK: string;
+
+        static FONT_SANS_8_WHITE: string;
+        static FONT_SANS_16_WHITE: string;
+        static FONT_SANS_32_WHITE: string;
+        static FONT_SANS_64_WHITE: string;
+        static FONT_SANS_128_WHITE: string;
+
+        // Edge Handling
+        static EDGE_EXTEND: 1;
+        static EDGE_WRAP: 2;
+        static EDGE_CROP: 3;
+
+        // Properties
+        bitmap: Bitmap;

```

Comment:
can you move this line back below the constructors? this apparently fixed something #393

Reply:
I'll take a look at it

Reply:
I created a typescript project using my forks version as a dependency and I can't reproduce the issue you mentioned, the issue was probably something else. Therefore moving the statics is a purely stylistic matter. Should I move it anyway?